packages/auth/tsconfig.json

```diff
@@ -0,0 +1,47 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "esnext",
+    "lib": ["dom", "esnext"],
+    "jsx": "react",
+    "importHelpers": true,
+    "declaration": true,
+    "sourceMap": true,
+    "moduleResolution": "node",
+    "forceConsistentCasingInFileNames": true,
+    "strict": true,
+    "pretty": true,
+    "removeComments": true,
+    "noImplicitAny": true,
+    "strictFunctionTypes": true,
+    "noImplicitThis": true,
+    "alwaysStrict": true,
+    "stripInternal": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "noImplicitReturns": true,
+    "noFallthroughCasesInSwitch": true,
+    "emitDecoratorMetadata": false,
+    "noEmit": false,
+    "experimentalDecorators": true,
+    "allowSyntheticDefaultImports": true,
+    "suppressImplicitAnyIndexErrors": false,
+    "outDir": "./dist",
+    "baseUrl": "./src",
+    "skipLibCheck": true,
+    "esModuleInterop": true,
+    "strictNullChecks": true,
+    "paths": {
+      "@stacks/auth": ["../../auth/src"],
+      "@stacks/cli": ["../../cli/src"],
+      "@stacks/common": ["../../common/src"],
+      "@stacks/encryption": ["../../encryption/src"],
+      "@stacks/keychain": ["../../keychain/src"],
+      "@stacks/network": ["../../network/src"],
+      "@stacks/profile": ["../../profile/src"],
+      "@stacks/storage": ["../../storage/src"],
+      "@stacks/transactions": ["../../transactions/src"]
+    }
+  },
+  "include": ["src/**/*", "tests/**/*"]
```

Comment:
I think this tsconfig would result in `test` files being output in the `/dist` directory and being published within the npm package.

Reply:
That doesn't seem to be the case, I wasn't able to find test files in the published packages.